app/models/grouping.rb

```diff
@@ -713,6 +713,37 @@ def self.get_groupings_for_assignment(assignment, user)
     end
   end
 
+  def self.get_assign_scans_grouping(assignment, user)
+    if assignment.scanned_exam?
+      grouping = nil
+      if user.ta?
```

Comment:
I already said TAs cannot do this. You can remove this part, and the `user` argument.